lib/services/livesync/livesync-service.ts

```diff
@@ -91,6 +91,8 @@ export class LiveSyncService extends EventEmitter implements IDebugLiveSyncServi
 				// In case we are stopping the LiveSync we must set usbLiveSyncService.isInitialized to false,
 				// as in case we execute nativescript-dev-typescript's before-prepare hook again in the same process, it MUST transpile the files.
 				this.$usbLiveSyncService.isInitialized = false;
+				// After stopping LiveSync we need to dispose of everything in order for the process to end and release all resources
+				this.$injector.dispose({ force: true });
```

Comment:
this will cause issues with device detection in Sidekick where we have a long living process. We shouldn't dispose the injector in this case, as this will kill iOS devices detection as well. We should find a better general solution for this case.